lib/msf/core/post/linux/kernel.rb

```diff
@@ -187,7 +187,8 @@ def exec_shield_enabled?
   # @return [Boolean]
   #
   def unprivileged_bpf_disabled?
-    cmd_exec('cat /proc/sys/kernel/unprivileged_bpf_disabled').to_s.strip.eql? '1'
+    unprivileged_bpf_disabled = cmd_exec('cat /proc/sys/kernel/unprivileged_bpf_disabled').to_s.strip
```

Comment:
```suggestion
    unprivileged_bpf_disabled = read_file('/proc/sys/kernel/unprivileged_bpf_disabled').to_s.strip
```
I think using `read_file` instead of `cat` would be better because it also checks if the file is present and is readable. :)

Reply:
Using `read_file` here is more complicated.

You're welcome to update this method, and all others in this mix, to use `read_file` once this lands.